src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -86,12 +87,12 @@ public async Task<(Error, string login)> GetLoginByCommit(string repoOwner, stri
             }
             catch (RateLimitExceededException ex)
             {
-                _rateLimitError = Errors.GitHubApiFailed($"GET /repos/:owner/:repo/commits/:sha", ex);
+                _rateLimitError = Errors.GitHubApiFailed(apiFailedMessage, ex);
```

Comment:
Log messages are de-duped by content. Left side ensures this error only has one log entry.

Reply:
The original warning message doesn't contain enough information about what happened. Want to add some info here.

---
In reply to: [222884924](https://github.com/dotnet/docfx/pull/3467#discussion_r222884924) [](ancestors = 222884924)